content/700-optimize/700-known-limitations.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: 'Known Limitations'
+metaTitle: 'Optimize: Known Limitations'
+metaDescription: 'Learn about known limitations of Optimize.'
+tocDepth: 3
+toc: true
+---
+
+Below are the known limitations when using Prisma Optimize. If you are aware of any limitations that are missing, please let us know on the `#help-and-questions` channel in our community [Discord](https://pris.ly/discord).
+
```

Comment:
If they are using Optimize Extension with Accelerate Extension then the Accelerate extension should be extended at the end.

I assume there was some conversation related to it, we can document that here in Known Limitations section